pyspecProcScripts/phasing.py

```diff
@@ -129,233 +149,220 @@ def ph1_real_Abs(s,dw,ph1_sel=0,ph2_sel=1,fl = None):
         automated phase correction of NMR spectra. Journal of Magnetic
         Resonance (San Diego, Calif. : 1997), 201(2), 230–238.
         https://doi.org/10.1016/j.jmr.2009.09.017
-    '''
-    if fl: fl.push_marker() 
-    ph1 = nddata(r_[-5:5:70j]*dw,'phcorr')
-    dx = np.diff(ph1.getaxis('phcorr')[r_[0,1]]).item()
-    ph1 = np.exp(-1j*2*pi*ph1*s.fromaxis('t2'))
+    """
+    if fl:
+        fl.push_marker()
+    ph1 = nddata(r_[-5:5:70j] * dw, "phcorr")
+    dx = np.diff(ph1.getaxis("phcorr")[r_[0, 1]]).item()
+    ph1 = np.exp(-1j * 2 * pi * ph1 * s.fromaxis("t2"))
     s_cost = s * ph1
-    ph0 = s_cost.C.sum('t2')
+    ph0 = s_cost.C.sum("t2")
     ph0 /= abs(ph0)
     s_cost /= ph0
     if fl:
-        fl.next('phasing cost function')
-    s_cost.run(np.real).run(abs).sum('t2')
-    s_cost = s_cost['ph2',1]['ph1',0]
+        fl.next("phasing cost function")
+    s_cost.run(np.real).run(abs).sum("t2")
+    s_cost = s_cost["ph2", 1]["ph1", 0]
     if fl:
-        fl.plot(s_cost,'.')
-    s_cost.smoosh(['vd','phcorr'],'transients')
-    ph1_opt = np.asarray(s_cost.argmin('transients').item())
-    logging.info(strm("THIS IS PH1_OPT",ph1_opt))
-    logging.info(strm('optimal phase correction',repr(ph1_opt)))
+        fl.plot(s_cost, ".")
+    s_cost.smoosh(["vd", "phcorr"], "transients")
+    ph1_opt = np.asarray(s_cost.argmin("transients").item())
+    logging.debug(strm("THIS IS PH1_OPT", ph1_opt))
+    logging.debug(strm("optimal phase correction", repr(ph1_opt)))
     # }}}
     # {{{ apply the phase corrections
-    def applyphase(arg,ph1):
-        arg *= np.exp(-1j*2*pi*ph1*arg.fromaxis('t2'))
-        ph0 = arg.C.sum('t2')
+    def applyphase(arg, ph1):
+        arg *= np.exp(-1j * 2 * pi * ph1 * arg.fromaxis("t2"))
+        ph0 = arg.C.sum("t2")
         ph0 /= abs(ph0)
         arg /= ph0
         return arg
+
     def costfun(ph1):
-        logging.info(strm("PH1 IS THIS:",ph1))
+        logging.debug(strm("PH1 IS THIS:", ph1))
         if type(ph1) is np.ndarray:
             ph1 = ph1.item()
         temp = s.C
-        retval = applyphase(temp,ph1).run(np.real).run(abs).sum('t2').item()
+        retval = applyphase(temp, ph1).run(np.real).run(abs).sum("t2").item()
         return retval
-    r = minimize(costfun,
-            ph1_opt,
-            bounds = [(ph1_opt-dx,ph1_opt+dx)])
+
+    r = minimize(costfun, ph1_opt, bounds=[(ph1_opt - dx, ph1_opt + dx)])
     assert r.success
-    s = applyphase(s,r.x.item())
-    fl.plot(r.x,r.fun,'x')
+    s = applyphase(s, r.x.item())
+    fl.plot(r.x, r.fun, "x")
```

Comment:
here and above, I had just run black and gotten rid of excessive info  statements.